roles/user/handlers/main.yml

```
@@ -1,10 +1,10 @@
 ---
 # handlers file for user
 - name: Update desktop menu
-  command:
+  ansible.builtin.command:
     /usr/bin/update-desktop-database
 - name: Suggest restart
-  command: >-
+  ansible.builtin.command: >-
     notify-send -u critical "JMU Software Change"
     "Changes have been made to your machine that
     will only take effect when you log out or
```

```suggestion
  ansible.builtin.command:
    cmd: >-
      notify-send -u critical "JMU Software Change"
      "Changes have been made to your machine that
      will only take effect when you log out or
```

And GitHub won't let me comment it but the next line would need to be indented to...